src/library.jsx

```diff
@@ -0,0 +1,64 @@
+
+const Box = (props) => {
+    var width = "";
+    var height = "";
+
+    switch(props.width){
+        case "sm":
+            width = "100px"
+            break;
+        case "md":
+            width = "250px"
+            break;
+        case "lg":
+            width = "500px"
+            break;
+    }
+
+    switch(props.height){
+        case "sm":
+            height = "100px";
+            break;
+        case "md":
+            height = "200px";
+            break;
+        case "lg":
+            height = "300px";
+            break;
+    }
+
+    return (
+        <div style={{width: width, height: height, backgroundColor:"lightcyan", color:"black", padding: "10px"}}>{props.children}</div>
```

Comment:
I wanted the ability to override the background color but also enforce a default. I can see it wasn't clear in the assignment instructions so we're good here.